ArkhamOverlay/Pages/Overlay/StatDisplay.xaml

```diff
@@ -16,10 +16,13 @@
              Visibility ="{Binding StatTrackingVisibility}">
     <Border.Resources>
         <BooleanToVisibilityConverter x:Key="BoolToVisibility" />
-        <Style x:Key="StatText" TargetType="{x:Type TextBlock}">
+        <Style x:Key="StatText" TargetType="{x:Type utils:OutlinedTextBlock}">
             <Setter Property="FontSize" Value="{Binding DataContext.OverlayFontSize, RelativeSource={RelativeSource AncestorType=Window}}" />
             <Setter Property="Text" Value="{Binding Name}" />
-            <Setter Property="Foreground" Value="{Binding PlayerNameBrush}" />
+            <!--<Setter Property="FontWeight" Value="Bold" />-->
```

Comment:
nit: remove unused property or uncomment it

Reply:
done